app/src/main/res/layout/fragment_tracking_protection_blocking.xml

```diff
@@ -75,7 +71,17 @@
             android:clickable="true"
             android:focusable="true"
             app:categoryItemDescription="@string/etp_tracking_content_description"
-            app:categoryItemIcon="@drawable/ic_tracking_content"
             app:categoryItemTitle="@string/etp_tracking_content_title" />
+
+        <org.mozilla.fenix.trackingprotection.TrackingProtectionCategoryItem
+            android:id="@+id/category_redirect_trackers"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:background="?android:attr/selectableItemBackground"
+            android:clickable="true"
+            android:focusable="true"
+            android:visibility="invisible"
```

Comment:
Usually we use `gone` because if something got added below this, it would take up space.